### ISSUE TITLE
Workshop: Database Seeding

### ISSUE THREAD

**Issue by lamara11** [amosproj/amos2024ss01-xcelerator-demo-app#138]
## User story

1. As a developer
2. I want to host a workshop
3. So that other developers can learn about the seeding of the database and get frontend and backend running at the same time
## Acceptance criteria

- [x] Max and Jonas to host a workshop
- [x] The workshop teaches developers about the seeding of the database and get frontend and backend running at the same time

## Definition of done (DoD)

- [x] All acceptance criteria were met
- [x] Pipeline is building successfully
- [x] Unit/integration tests are passing
- [x] Code has been reviewed and merged into the primary development branch

## DoD general criteria

- [x] Developers agreed to release
- [x] Product owner approved features
- [x] Feature has been fully implemented

**Comment by Sungonogi** [comment]
I proposed this work shop because right now after #102 was merged only David, Jonas and maybe Sasa are able to build everything on the develop branch and actually see facilities. A detailed description of how to get the backend and the database to run would also work but it is necessary for us other devs to know this. After already trying and getting help by David a little I can say it includes at least:
- Distribute the .env file so everyone has it
- show us the Prisma commands (prisma:generate, prisma:seed)
- getting docker-compose up to work and then show us which containers to run using docker and which ones to run in the terminal
- finding a way for it to work on windows (everyone but David and Jonas uses Windows)
